README.md

```diff
@@ -31,31 +31,31 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+DOM, stands for the Document Object Model. This is the thing that gives javascript access to the HTML.
 2. What is an event?
-
+Anything that happens on the web page which will change the Document.
 3. What is an event listener?
-
+An event listener is something that will tell an event to fire, causing javascript to change the DOM.
 4. Why would we convert a NodeList into an Array?
-
+If you change a NodeList into an Array you get access to all the nifty Array methods.
 5. What is a component?
-
+Essentially its an Object/Class/something you can refer to, so that it can do nifty things int the future.
```

Comment:
This is how I answered this one: ```A component is the intertwined html/css/js that is used to make objects on the window smarter. A component is said to be the intertwined use of html/css/js because, the js code that makes the window smarter refers to both the html and css of the window to make an object do what you want it to do (e.g. adding/removing a class name to a class in js in order to add/remove a specific style in the css.)```